test/theme_test.go

```diff
@@ -5,28 +5,92 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
 
 	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/theme"
 )
 
-func Test_NewTheme_checkForUniqueColors(t *testing.T) {
-	th := NewTheme().(*configurableTheme)
-	seen := map[string]fyne.ThemeColorName{}
-	for cn, c := range th.colors {
-		r, g, b, a := c.RGBA()
-		key := fmt.Sprintf("%d %d %d %d", r, g, b, a)
-		assert.True(t, seen[key] == "", "color value %#v for color %s already used for color %s in theme %s", c, cn, seen[key], th.name)
-		seen[key] = cn
+// Try to keep these in sync with the existing color names at theme/colors.go.
+var knownColorNames = []fyne.ThemeColorName{
+	theme.ColorNameBackground,
+	theme.ColorNameButton,
+	theme.ColorNameDisabledButton,
+	theme.ColorNameDisabled,
+	theme.ColorNameError,
+	theme.ColorNameFocus,
+	theme.ColorNameForeground,
+	theme.ColorNameHeaderBackground,
+	theme.ColorNameHover,
+	theme.ColorNameHyperlink,
+	theme.ColorNameInputBackground,
+	theme.ColorNameInputBorder,
+	theme.ColorNameMenuBackground,
+	theme.ColorNameOnPrimary,
+	theme.ColorNameOverlayBackground,
+	theme.ColorNamePlaceHolder,
+	theme.ColorNamePressed,
+	theme.ColorNamePrimary,
+	theme.ColorNameScrollBar,
+	theme.ColorNameSelection,
+	theme.ColorNameSeparator,
+	theme.ColorNameShadow,
+	theme.ColorNameSuccess,
+	theme.ColorNameWarning,
+}
+
+// Try to keep this in sync with the existing variants at theme/theme.go
```

Comment:
Lets add a comment in that file as well to keep this in sync. See comment above.